code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -1,5 +1,21 @@
-/obj/machinery/reagentgrinder
 
+
+var/global/list/juice_items = list (
```

Comment:
Yeah, it does both prioritizing juice. The only item that has both a grind and juice special output is ketchup/tomatojuice, so I'm going to make ketchup a tomatojuice recipe. This is essentially a revival of my (years old) grinder refactor, but done right.